californium-core/src/main/java/org/eclipse/californium/core/observe/ObserveNotificationOrderer.java

```diff
@@ -83,14 +83,27 @@ public int getCurrent() {
 	}
 
 	/**
-	 * Returns true if the specified notification is newer than the current one.
+	 * Check, if the provided notification is newer than the current one.
 	 * 
 	 * @param response the notification
-	 * @return true if the notification is new
+	 * @return {@code true} if the notification is new, or the response is no
+	 *         notify
 	 */
-	public synchronized boolean isNew(Response response) {
+	public boolean isNew(Response response) {
+		return isNew(response.getOptions().getObserve());
+	}
+
+	/**
+	 * Check, if the provided value of the observer option is newer than the
+	 * current one.
+	 * 
+	 * @param observe the value of the observe option. Maybe {@code null}, if
+	 *            the observation is canceled
+	 * @return true if the notification is new, or canceled
+	 * @since 2.3
```

Comment:
2.2.1 or 2.3 ?

(new API so 2.3 ? or maybe this part of the code in considering as internal)